packages/frontend/src/components/ProjectConnection/ProjectConnectFlow/ConnectSuccess.tsx

```diff
@@ -52,7 +52,7 @@ const ConnectSuccess: FC<ConnectSuccessProps> = ({ projectUuid }) => {
 
                             const rect = el.getBoundingClientRect();
 
-                            confetti({
+                            void confetti({
```

Comment:
didn't know confetti was a promise. now I know!

Reply:
"I promise you will receive confetti", or more cynically, "I promise you'll be finding bits of confetti throughout the house over the next few months"